vcx/wrappers/java/demo/buildlibs.sh

```diff
@@ -0,0 +1,39 @@
+current_dir=$(pwd)
+source ~/.profile
+
+echo "######Building LIBVCX######"
+cd ../../../libvcx
+cargo build
+echo $(pwd)
+
+echo "#####Building LibIndy#####"
+cd ../../libindy
+cargo build
+echo $(pwd)
+
+echo "#####Building LibNullPay####"
+cd ../libnullpay
+cargo build
+echo $(pwd)
+
+echo "####Building VCX JAR#####"
+echo $current_dir
+cd $current_dir
+cd ..
+./gradlew build
+
+cd $current_dir
+
+
+
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+        sudo cp ../../../libvcx/target/debug/libvcx.so /usr/lib/libvcx.so
```

Comment:
I suggest to use env variables instead of moving artifacts to system directories

Reply:
which env variable I should use?

Reply:
let me share context and reason of my suggestion to be more clear.
This PR introduce demo in java wrapper. And currently we are discussing a script in this demo called  `buildlibs.sh`
But in fact this script perform installation of manual built of libindy and libnullpay to *system* path (/usr/lib). It may be unexpected by user and damage his environment, like replace system stable version of libindy by something unstable or even experimental or broken.
So main point from my side is to avoid actual installation libraries to system path, but rather to use some env variable to make it available for demo, or copy it to local folder, or give a chance to user to choose the behavior.